### PR TITLE
View past access requests in Manage User table

### PR DESCRIPTION
**Pull request by mauberti-bc**
## Links to Jira Tickets

- N/A

## Description of Changes

- Show pending, approved and rejected requests on the Manage Users page
- Allow the details of approved and rejected requests to be viewed
- Add access requests to the database seed
- Reorganizes Manage User-related files in the app

### CODE REVIEW

File: app/src/constants/colours.ts [hunk 1]
```diff
@@ -65,6 +65,16 @@ const NRM_REGION_COLOUR_MAP = {
   'Skeena Natural Resource Region': { colour: red }
 };
 
+/**
+ * Colour map for access request chips.
+ *
+ */
+const ACCESS_REQUEST_STATUS_COLOUR_MAP = {
```
Comment (MacDeluca) [inline app/src/constants/colours.ts:72]: praise: Hell yeah! 
This is the correct way for these types of values, instant lookup vs having to traverse through an array of values.

File: database/src/seeds/03_basic_project_survey_setup.ts [hunk 2]
```diff
@@ -44,6 +46,11 @@ export async function seed(knex: Knex): Promise<void> {
     `);
   }
 
+  // Insert access requests
+  for (let i = 0; i < 15; i++) {
+    await knex.raw(`${insertAccessRequest()}`);
```
Comment (MacQSL) [inline database/src/seeds/03_basic_project_survey_setup.ts:51]: mega nit: Promise.all to improve seed execution time

Comment (mauberti-bc) [reply database/src/seeds/03_basic_project_survey_setup.ts:51]: Fixed

**Review by MacDeluca — DISMISSED** [review]
Tested locally and works great! 🧄

**Review by MacDeluca — DISMISSED** [review]
🆎

**Review by NickPhura — APPROVED** [review]

**Pull request merged**